doc/source/data/doc_code/quick_start.py

```diff
@@ -1,64 +1,119 @@
 # flake8: noqa
 # fmt: off
 
-# __data_setup_begin__
-
+# __create_from_python_begin__
 import ray
 
-# Create a Dataset of Python objects.
+# Create a Dataset from Python objects, which are also held as Python objects.
```

Comment:
This technically isn't creating a Dataset _from_ Python objects, it's lazily generating Python objects.
```suggestion
# Generate a Dataset of Python objects.
```